src/components/Layout.jsx

```diff
@@ -1,14 +1,16 @@
-import React from 'react';
 import Addcleaner from './Addcleaner';
 import Nav from './Nav';
+import CleanerDetails from './cleanerDetails';
 // import Home from './Home';
 
 const Layout = () => (
   <div className="container">
     <Nav />
     {/* <Home /> */}
```

Comment:
In order to make your code more readable, don't forget to remove the comment when you are done.